packs/pagerduty/actions/resolve_incident.yaml

```diff
@@ -0,0 +1,10 @@
+name: resolve_incident
+runner_type: run-python
+description: Resolve an incident whose key is provided
+enabled: true
+entry_point: resolve_incident.py
+parameters:
+  event_keys:
+    type: array
```

Comment:
I tested it by supplying comma seperated keys, and it runs fine. 
like for instance : 
 st2 run pagerduty.ack_incident event_key=21ab8308116,6ff46b1abbe9ee82965e764